pysatCDAAC/instruments/cosmic2_ivm.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+"""Supports IVM data from the COSMIC2 satellite.
+
+The Constellation Observing System for Meteorology, Ionosphere, and Climate
+(COSMIC) is comprised of six satellites in LEO with GPS receivers. The
+occultation of GPS signals by the atmosphere provides a measurement of
+atmospheric parameters. Data downloaded from the COSMIC Data Analaysis
+and Archival Center.
+
+More info about the dataset can be found at https://doi.org/10.5065/t353-c093
+
+Properties
+----------
+platform
+    'cosmic2'
+name
+    'ivm'
+tag
+    None supported
+inst_id
+    ['e1', 'e2', 'e3', 'e4', 'e5', 'e6']
+
+Warnings
+--------
+- Routine was not produced by COSMIC team
+
+"""
+
+import datetime as dt
+import functools
+import os
+import warnings
+
+import pysat
+from pysat.instruments.methods import general as mm_gen
+from pysatCDAAC.instruments.methods import general as mm_cdaac
+
+# ----------------------------------------------------------------------------
+# Instrument attributes
+
+platform = 'cosmic2'
+name = 'ivm'
+
+tags = {'': 'Ion Velocity Meter data'}
+
+inst_ids = {'e1': [''], 'e2': [''], 'e3': [''], 'e4': [''], 'e5': [''],
+            'e6': ['']}
+
+# Because all data products are stored in one tar file, inst_id not used
+directory_format = os.path.join('{platform}', '{name}', '{tag}')
```

Comment:
Add inst_id here.

Reply:
If inst_id is added to the file organization then the directory_format line may be removed. It is there to account for the non-standard layout.